setup.py

```diff
@@ -19,7 +19,7 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub==1.25.2',
+        'PyGithub==1.25.2-squash',
```

Comment:
I don't think we have done it in the setup.py before, but this link seems to indicate that it's possible:
http://mike.zwobble.org/2013/05/adding-git-or-hg-or-svn-dependencies-in-setup-py/
```
setup(
    install_requires=[
        "mayo==0.2.1"
    ],
    dependency_links=[
        "git+ssh://git@github.com/mwilliamson/mayo.git@0.2.1#egg=mayo-0.2.1"
    ]
)
```